project/src/func.cpp

```diff
@@ -23,8 +23,8 @@ const std::string kUserFollower = "user_follower:";
 const std::string kWarblePost = "warble_post:";
 const std::string kWarbleChildren = "warble_children:";
 
-// keeps track of id of warbles posted
-int latest_warble_id = 0;
+// prefix used to read and write latest warble from kvstore
+const std::string kLatestWarbleString = "latest_warble_id:";
```

Comment:
This statement appears in multiple places, which means if I want to change it, I will have to change in multiple places. I think it would be better to place it in a *.h file and include the header file.

Reply:
thanks, fixed. Constants are now in project/src/kv_tags.h